skops/card/_model_card.py

```diff
@@ -380,8 +380,15 @@ class Card:
         ``Path``/``str`` of the model or the actual model instance that will be
         documented. If a ``Path`` or ``str`` is provided, model will be loaded.
 
-    model_diagram: bool, default=True
-        Set to True if model diagram should be plotted in the card.
+    model_diagram: bool or str, default=True
+        If using the skops template, setting this to ``True`` will add the model
+        diagram, as generated by sckit-learn, to the default section. Passing a
```

Comment:
I think we should mention what the name of the section is, for them to be able to select and modify it later.